tuf/keys.py

```diff
@@ -1232,8 +1266,8 @@ def encrypt_key(key_object, password):
  
   # check_crypto_libraries() should have fully verified _GENERAL_CRYPTO_LIBRARY.
   else: # pragma: no cover
-    message = 'Invalid crypto library: ' + repr(_GENERAL_CRYPTO_LIBRARY) + '.'
-    raise tuf.UnsupportedLibraryError(message) 
+    raise tuf.UnsupportedLibraryError('Invalid crypto library:'
+      ' ' + repr(_GENERAL_CRYPTO_LIBRARY) + '.') 
```

Comment:
I wanted to address comment [#1](https://github.com/theupdateframework/tuf/pull/289#issuecomment-171392644) in a separate pull request, so intentionally left the large block of exception messages in check_crypto_libraries() untouched (which might change if we modify how we import some of these crypto modules).

I'll update this block in this pull request instead.